.github/workflows/build_all.yml

```diff
@@ -22,7 +22,8 @@ on:
       - master
       - 'v1.*'
     types: [ opened, reopened, synchronize ]
-
+env:
+  LF_ARTIFACTORY_BASE_URL: "https://artifactory.magmacore.org:443/artifactory/"
```

Comment:
All other repository definitions are within the respective jobs.
So for consistency, this should
- either also go into the respective job, or
- the other occurrences of artifactory URLs should be refactored to also use the global definition.